racer.el

```diff
@@ -460,6 +460,18 @@ Commands:
   (cons (get-text-property 0 'file arg)
         (get-text-property 0 'line arg)))
 
+(defun racer-get-bol-to-point ()
+  "Get text from start of line to point."
+  (let ((p (point)))
+    (save-excursion
+      (beginning-of-line)
+      (let ((bol (point)))
+        (buffer-substring-no-properties bol p)))))
+
+(defun racer-current-column ()
```

Comment:
For private functions (not used outside of racer), we use two hyphens. Could we call this `racer--current-column`?